src/app/containers/MediaPlayer/index.jsx

```diff
@@ -1,55 +1,51 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import pathOr from 'ramda/src/pathOr';
-import Caption from '../Caption';
-import videoMetadata from './metadata';
-import {
-  NestedGridParentLarge,
-  NestedGridParentSmall,
-  NestedGridItemChildSmall,
-  NestedGridItemChildLarge,
-  GridItemConstrainedLargeNoMargin,
-  GridItemConstrainedSmall,
-} from '../../lib/styledGrid';
-
+import styled from 'styled-components';
+import Amp from './Amp';
+import Canonical from './Canonical';
```

Comment:
The naming convention in use elsewhere (well, Chartbeat and ATI) is `./amp` (for directory name) and `AmpChartbeatBeacon` (for component name), can we try to standardise? I don't mind what the standard is so long as it's fairly predictable. Happy for this to be done in a separate PR though.